contents/dataframes_missing.md

```diff
@@ -0,0 +1,196 @@
+## Missing Data {#sec:missing}
+
+Let's dive into how to **handle missing values** in `DataFrames.jl`.
+We'll cover **three main approaches for dealing with missing data**:
+
+1. **filtering missing** values with `ismissing` and `dropmissing`
+1. **filling or replacing missing** values with `coalesce`
+1. **skipping over missing** values with `skipmissing`
+
+First, we need a `DataFrame` filled  with missing values to showcase these approaches:
+
+```jl
+sco("""
+df_missing = DataFrame(;
+    name=[missing, "Sally", "Alice", "Hank"],
+    age=[17, missing, 20, 19],
+    grade_2020=[5.0, 1.0, missing, 4.0],
+)
+"""; process=without_caption_label)
+```
+
+This is ther same `DataFrame` from @sec:dataframes but with some `missing` values added.
+
+### `Missing` type {#sec:missing_type}
+
+Some languages have several types to represent missing values.
+One such example is R which uses `NA`, `NA_integer_`, `NA_real_`, `NA_character_`, and `NA_complex_`.
+**Julia**, on the contrary, has **only one**: `Missing`.
```

Comment:
what about `NaN` and `nothing`?

Reply:
This is missing values, `NaN` and `Nothing` are other kinds of types. Shall we include a `NOTE` explaining those. I think that it would introduce more confusion than enlightenment here.

Reply:
then, how different is the Julia NaN from the ones you mentioned for R?, maybe a note will be appropriate.

Reply:
The ones mentioned are `NA`s in R. Non-Available. This is how R represents missing. `NaN` is different in R.

Reply:
Are you sure that those `NA` types are used by R base to represent missing values?

Reply:
Yes, I am.
https://stackoverflow.com/a/70472074